### DISCUSSION TITLE
Hire someone to make me a Helicopter FDM for JSBSim?

### DISCUSSION THREAD

**Discussion by PeterJenkLab** [JSBSim-Team/jsbsim#739] [General]
Hello All

Is someone interested in doing a FDM for JSBsim for use in the Unreal Engine.
OR can someone direct me where I can maybe hire someone or send me in the right direction.
I have the 3d model.
Its for a VR project.

The helicopter is a
Sikorsky SH-3 Sea King

If interested please message me your rates or if you want to chat more about the job
I know doing a Helicopter FDM for JSBsim is hard to do, that's why I need someone with more experience using it.

Thanks for your time in advanced.

**Comment by bomber64** [comment]
Mmmmmmmmmmmmm
